shopwareerpconnect/invoice.py

```diff
@@ -25,7 +25,7 @@
 from openerp.addons.connector.queue.job import job, related_action
 from openerp.addons.connector.unit.synchronizer import Exporter
 from openerp.addons.connector.event import on_record_create
-from openerp.addons.connector_ecommerce.event import (on_invoice_paid,
+from openerp.addons.connector_ecommerce.models.event import (on_invoice_paid,
                                                       on_invoice_validated)
 from openerp.addons.connector.exception import IDMissingInBackend
```

Comment:
should be odoo. not openerp. everywhere! This is v9 onwards

Reply:
Yes in from version 9 it is odoo not openerp.

Reply:
No sir it is "openerp" in version 9 from 10 it changes to "odoo", I have checked it in base's modules and few others they all have "openerp" in import line.